tla/consensus/ccfraft.tla

```diff
@@ -142,14 +142,22 @@ VARIABLE removedFromConfiguration
 RemovedFromConfigurationTypeInv ==
     removedFromConfiguration \subseteq Servers
 
+VARIABLE hasJoined
```

Comment:
Can we add a comment on this/remove the now-inaccurate comment from `removedFromConfiguration`?

Reply:
Done